src/components/ProjectionResult.js

```diff
@@ -1,52 +1,118 @@
-import { useSelector } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
 import { Divider, Form, Header, Tab } from 'semantic-ui-react';
 
+import { setRecommendationMode } from '../store/actions';
 import Recommendation from './Recommendation';
 
 function ProjectionResult() {
-  const { projectionLocation } = useSelector(state => ({
+  const dispatch = useDispatch();
+  const { projectionLocation, location } = useSelector(state => ({
     projectionLocation: state.projectionLocation,
+    location: state.location,
   }));
   const { t, i18n } = useTranslation();
 
+  const [future, setFuture] = useState(false);
+
   const panes = [
     {
-      menuItem: t('tab.scenario1'),
-      render: () => <Tab.Pane>Coming soon...</Tab.Pane>,
+      menuItem: t('recommendationMode.today'),
+      recommendationMode: 'today',
+      render: () => (
+        <Tab.Pane>
+          {' '}
+          <Header>
+            {`${location.forestType} - ${translate(
+              'forestType',
+              location.forestType,
+              i18n.language,
+            )}`}
+          </Header>
+          <Divider hidden />
+          <Recommendation forestTypeToday={location.forestType} />
+        </Tab.Pane>
+      ),
     },
     {
-      menuItem: t('tab.scenario2'),
+      menuItem: t('recommendationMode.moderate'),
+      recommendationMode: 'moderate',
       render: () => (
         <Tab.Pane>
           <Header>
-            {translate(
+            {`${projectionLocation.forestType} - ${translate(
               'forestType',
               projectionLocation.forestType,
               i18n.language,
-            )}
+            )}`}
           </Header>
           <Form>
             <Form.Field>
-              <Form.Radio label="bereits heute mögliche Baumarten" checked />
+              <Form.Radio
+                label="bereits heute mögliche Baumarten"
+                name="radioGroup"
+                checked={future === false}
+                onChange={() => setFuture(false)}
+              />
             </Form.Field>
             <Form.Field>
               <Form.Radio
                 label="in Zukunft zusätzliche Baumarten"
-                checked={false}
+                name="radioGroup"
+                checked={future === true}
+                onChange={() => setFuture(true)}
               />
             </Form.Field>
           </Form>
           <Divider hidden />
-          <Recommendation forestType={projectionLocation.forestType} />
+          <Recommendation
+            forestTypeToday={location.forestType}
+            forestTypeFuture={projectionLocation.forestType}
+            future={future}
+          />
         </Tab.Pane>
       ),
     },
     {
-      menuItem: t('tab.scenario3'),
-      render: () => <Tab.Pane>Coming soon...</Tab.Pane>,
+      menuItem: t('recommendationMode.extreme'),
+      recommendationMode: 'extreme',
+      render: () => (
+        <Tab.Pane>
```

Comment:
Instead of duplication we should create a new component which will be reused for all panes. I'll let you decide if we need a specific component for "today".

Reply:
Good idea.